vscode-dotnet-runtime-library/src/test/unit/TestUtility.ts

```diff
@@ -149,6 +149,10 @@ export async function getLinuxSupportedDotnetSDKVersion(context: IAcquisitionWor
         {
             return '6.0.100';
         }
+        if (distroInfo.version < '22.06')

```

Comment:
This is a very narrow range. It seems strange to me that you'd want 9.0.100 (the later version) on 22.04 (the earlier version) compared with 8.0.100 on 22.06. Why is that?

Reply:
Thank you for looking at this so thoroughly -- I hope this helps. The dotnet support for these minor versions seems to be different, as is the dotnet thats installed on these CI machines. When we get a different CI machine, either in dneng internal or public, it was failing with 22.05 because that expects .NET 9,  but the 22.04 machines expect .NET 8. If I fix that version but not 22.04, it will make the other machines fail.